server/container/guice/memory-guice/src/main/java/org/apache/james/modules/mailbox/MemoryMailboxModule.java

```diff
@@ -94,7 +94,7 @@ protected void configure() {
         bind(InMemoryMailboxManager.class).in(Scopes.SINGLETON);
         bind(UnionMailboxACLResolver.class).in(Scopes.SINGLETON);
         bind(SimpleGroupMembershipResolver.class).in(Scopes.SINGLETON);
-        bind(MessageId.Factory.class).in(Scopes.SINGLETON);
+        bind(DefaultMessageId.Factory.class).in(Scopes.SINGLETON);
```

Comment:
Shouldn't the MessageId capabilities also be implemented ?

(another ticket...)